libevm/VM.cpp

```diff
@@ -815,7 +848,68 @@ void VM::interpretCases()
 			m_sp -= 2;
 		CASE_END
 
+#ifdef EVM_JUMPS_AND_SUBS
+		CASE_BEGIN(JUMPTO)
+			ON_OP();
+			updateIOGas();
+			m_pc = decode_jump_dest(m_code, m_pc);
```

Comment:
Have you handled out of bound code access issue somewhere else (like with push data)?